modules/lang-mustache/src/main/java/org/elasticsearch/script/mustache/RestSearchTemplateAction.java

```diff
@@ -71,9 +71,9 @@ public RestChannelConsumer prepareRequest(RestRequest request, NodeClient client
         SearchRequest searchRequest = new SearchRequest();
         RestSearchAction.parseSearchRequest(
             searchRequest,
-            request,
+            request.getRestApiVersion(),
```

Comment:
Avoid to use a RestRequest on parameters to parseSearchRequest. Now we need to include the RestApiVersion and the request params separately.